test/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -7,19 +7,19 @@ Object {
     "2339",
     "--write",
   ],
-  "cwd": "cases\\\\addMissingMember",
+  "cwd": "cases/addMissingMember",
```

Comment:
I was surprised to see these backslashes still here after the `normalizeSlashes` code 🤔